packages/api/tests/endpoints/test-pdrs.js

```diff
@@ -241,3 +289,93 @@ test('DELETE handles the case where the PDR exists in DynamoDb but not in S3', a
   const parsedBody = response.body;
   t.is(parsedBody.detail, 'Record deleted');
 });
+
+test('DELETE removes a PDR from RDS and DynamoDB', async (t) => {
+  // Create the same PDR in Dynamo and PG
+  const newDynamoPdr = fakePdrFactory('completed');
+  const pdrName = newDynamoPdr.pdrName;
+  const newPGPdr = fakePdrRecordFactory({
+    name: pdrName,
+    status: 'completed',
+    collection_cumulus_id: t.context.collectionCumulusId,
+    provider_cumulus_id: t.context.providerCumulusId,
+  });
+
+  // create a new PDR in Dynamo
+  await pdrModel.create(newDynamoPdr);
+
+  // create a new PDR in RDS
+  await pdrPgModel.create(t.context.knex, newPGPdr);
+
+  const response = await request(app)
+    .delete(`/pdrs/${pdrName}`)
+    .set('Accept', 'application/json')
+    .set('Authorization', `Bearer ${jwtAuthToken}`);
+  t.is(response.status, 200);
+
+  // Check Dynamo and RDS. The PDR should have been removed from both.
+  await t.throwsAsync(
+    pdrModel.get({ pdrName }),
+    { instanceOf: RecordDoesNotExist }
+  );
+
+  t.is(
+    await pdrPgModel.exists(t.context.knex, { name: pdrName }),
+    false
+  );
```

Comment:
Unbearably small nit: I think you could do

```
t.false(await pdrPgModel.exists(t.context.knex, { name: pdrName }))
```
if you were so inclined.

Reply:
Even more ✨ unbearable ✨ , you could also avoid writing `t.context.knex` twice in the test by doing
```
const { knex } = t.context;
```